src/middlewares/getOneArticle.js

```diff
@@ -0,0 +1,36 @@
+import models from '../api/models';
+import statusCode from '../helpers/constants/status.codes';
+
+const { Article } = models;
+/**
+ * containing all user's model controllers (signup, login)
+ *
+ * @export
+ * @class getOneArticle
+ */
+class getOneArticle {
+  /**
+   * this is a middleware which checks if the user is registered in to our database.
```

Comment:
IS this the description of what's happening here @MCFrank16 ?

Reply:
that is laziness for generating a new js doc, lemme go change it